src/wp-content/themes/twentyfifteen/js/functions.js

```diff
@@ -5,130 +5,162 @@
+
+				// Re-sync expanded states from oldContainer.
+				params.oldContainer.find( '.dropdown-toggle.toggle-on' ).each(function() {
+					var containerId = $( this ).parent().prop( 'id' );
+					$( params.newContainer ).find( '#' + containerId + ' > .dropdown-toggle' ).triggerHandler( 'click' );
+				});
+			}
+		});
+	}
 
-	secondary = $( '#secondary' );
-	button = $( '.site-branding' ).find( '.secondary-toggle' );
+	secondary = document.getElementById( 'secondary' );
+	button = document.querySelector( '.site-branding .secondary-toggle' );
 
 	// Enable menu toggle for small screens.
 	( function() {
 		var menu, widgets, social;
-		if ( ! secondary.length || ! button.length ) {
+		if ( ! secondary || ! button ) {
 			return;
 		}
 
 		// Hide button if there are no widgets and the menus are missing or empty.
-		menu    = secondary.find( '.nav-menu' );
-		widgets = secondary.find( '#widget-area' );
-		social  = secondary.find( '#social-navigation' );
-		if ( ! widgets.length && ! social.length && ( ! menu.length || ! menu.children().length ) ) {
-			button.hide();
+		menu    = secondary.querySelector( '.nav-menu' );
+		widgets = secondary.querySelector( '#widget-area' );
+		social  = secondary.querySelector( '#social-navigation' );
+		if ( ! widgets && ! social && ( ! menu || ! menu.children.length ) ) {
+			button.style.display = 'none';
 			return;
 		}
 
-		button.on( 'click.twentyfifteen', function() {
-			secondary.toggleClass( 'toggled-on' );
-			secondary.trigger( 'resize' );
-			$( this ).toggleClass( 'toggled-on' );
-			if ( $( this, secondary ).hasClass( 'toggled-on' ) ) {
-				$( this ).attr( 'aria-expanded', 'true' );
-				secondary.attr( 'aria-expanded', 'true' );
+		button.addEventListener( 'click', function() {
+			toggleClass( secondary, 'toggled-on' );
+			window.dispatchEvent( new Event( 'resize' ) );
+			toggleClass( this, 'toggled-on' );
+			if ( -1 !== this.className.indexOf( 'toggled-on' ) && -1 !== secondary.className.indexOf( 'toggled-on' ) ) {
+				this.setAttribute( 'aria-expanded', 'true' );
+				secondary.setAttribute( 'aria-expanded', 'true' );
 			} else {
-				$( this ).attr( 'aria-expanded', 'false' );
-				secondary.attr( 'aria-expanded', 'false' );
+				this.setAttribute( 'aria-expanded', 'false' );
+				secondary.setAttribute( 'aria-expanded', 'false' );
 			}
 		} );
 	} )();
 
 	/**
 	 * Add or remove ARIA attributes.
 	 *
-	 * Uses jQuery's width() function to determine the size of the window and add
-	 * the default ARIA attributes for the menu toggle if it's visible.
+	 * Determine the size of the window and add the default ARIA attributes
+	 * for the menu toggle if it's visible.
 	 *
 	 * @since Twenty Fifteen 1.1
 	 */
 	function onResizeARIA() {
-		if ( 955 > $window.width() ) {
-			button.attr( 'aria-expanded', 'false' );
-			secondary.attr( 'aria-expanded', 'false' );
-			button.attr( 'aria-controls', 'secondary' );
+		if ( 955 > document.documentElement.clientWidth ) {
```

Comment:
Yoda for comparisons? Yikes. Let's take the opportunity to fix this:

```suggestion
		if ( document.documentElement.clientWidth < 955 ) {
```